data-processing-lib/src/data_processing/pure_python/transform_orchestrator.py

```diff
@@ -0,0 +1,95 @@
+# (C) Copyright IBM Corp. 2024.
+# Licensed under the Apache License, Version 2.0 (the “License”);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#  http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an “AS IS” BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################
+
+import time
+import traceback
+from datetime import datetime
+
+from data_processing.data_access import DataAccessFactoryBase
+from data_processing.pure_python import TransformTableProcessor
+from data_processing.transform import TransformConfiguration, TransformStatistics
+from data_processing.utils import get_logger
+
+
+logger = get_logger(__name__)
+
+
+def orchestrate(
+    data_access_factory: DataAccessFactoryBase,
+    transform_config: TransformConfiguration,
+) -> int:
+    """
+    orchestrator for transformer execution
+    :param data_access_factory: data access factory
+    :param transform_config: transformer configuration
+    :return: 0 - success or 1 - failure
+    """
+    start_ts = datetime.now().strftime("%Y-%m-%d %H:%M:%S")
+    logger.info(f"orchestrator started at {start_ts}")
```

Comment:
I suggest adding a name to TransformConfiguration or Transformer (it can be automatically discovered from the class name) and print it here.
f"orchestrator {name} started at {start_ts}"

Reply:
done